flow360/component/simulation/models/surface_models.py

```diff
@@ -112,83 +143,140 @@ class Rotational(Flow360BaseModel):
 
 
 class Wall(BoundaryBase):
-    """Replace Flow360Param:
-    - NoSlipWall
-    - IsothermalWall
-    - HeatFluxWall
-    - WallFunction
-    - SolidIsothermalWall
-    - SolidAdiabaticWall
+    """
+    :class:`Wall` class defines the Wall boundary conditions below based on the input:
```

Comment:
Are we exposing these old boundary condition values to users?

Reply:
good catch. we should not.